libs/ardour/vst_plugin.cc

```diff
@@ -193,6 +193,7 @@ VSTPlugin::set_parameter (uint32_t which, float newval, sampleoffset_t when)
 		} else {
 			cerr << "effSetBypass failed rv=" << rv << endl; // XXX DEBUG
```

Comment:
Correction: We also need an empty line here, to avoid that the next line aligns with the `// XXX` above.